internal/providers/terraform/aws/registry.go

```diff
@@ -51,6 +51,9 @@ var ResourceRegistry []*schema.RegistryItem = []*schema.RegistryItem{
 	getElasticsearchDomainRegistryItem(),
 	getELBRegistryItem(),
 	getFSxWindowsFSRegistryItem(),
+	getGlueJobRegistryItem(),
+	getGlueCrawlerRegistryItem(),
+	getGlueCatalogDatabaseRegistryItem(),
```

Comment:
Let try to keep the alphabetical order:
```suggestion
	getGlueCatalogDatabaseRegistryItem(),
	getGlueCrawlerRegistryItem(),
	getGlueJobRegistryItem(),
```